packages/sdk/app-framework/src/plugins/IntentPlugin/intent.ts

```diff
@@ -38,6 +38,7 @@ export type Intent = {
   /**
    * Any data needed to perform the desired action.
    */
+  // TODO(burdon): Typed intents.
```

Comment:
👍 

@ZaymonFC played with one possible way of typing intent data in arena that we could consider.

Reply:
https://github.com/dxos/arena-app/blob/main/src/lib/intent.ts

https://github.com/dxos/arena-app/blob/36557c4be31e16427b3627e8ceb4f41b03bf90a6/src/plugins/RoomManager/room-manager-plugin.tsx#L68